addons/mail/static/src/models/activity_view/activity_view.js

```diff
@@ -0,0 +1,104 @@
+/** @odoo-module **/
+
+import { registerModel } from '@mail/model/model_core';
+import { attr, many2one } from '@mail/model/model_field';
+
+registerModel({
+    name: 'mail.activity_view',
+    identifyingFields: ['activityBoxView', 'activity'],
+    lifecycleHooks: {
+        _created() {
+            this.onAttachmentCreated = this.onAttachmentCreated.bind(this);
+            this.onClickActivity = this.onClickActivity.bind(this);
+            this.onClickCancel = this.onClickCancel.bind(this);
+            this.onClickEdit = this.onClickEdit.bind(this);
+            this.onClickUploadDocument = this.onClickUploadDocument.bind(this);
+            this.onClickDetailsButton = this.onClickDetailsButton.bind(this);
+        }
+    },
+    recordMethods: {
+        /**
+         * @param {CustomEvent} ev
+         * @param {Object} ev.detail
+         * @param {mail.attachment} ev.detail.attachment
+         */
+        onAttachmentCreated(ev) {
+            this.activity.markAsDone({ attachments: [ev.detail.attachment] });
+        },
+
```

Comment:
Remove whitespace between items in a category/section of `registerModel`.
The guidelines is spacing when using `class` syntax, but with object there is no spacing in-between.